odxtools/basevariantpattern.py

```diff
@@ -16,7 +16,8 @@ class BaseVariantPattern(VariantPattern):
     """Base variant patterns are variant patterns used to identify the
     base variant of an ECU.
     """
-    matching_base_variant_parameters: list[MatchingBaseVariantParameter]
+    matching_base_variant_parameters: list[MatchingBaseVariantParameter] = field(
```

Comment:
Not a `NamedItemList` ?

Reply:
for whatever reason, MATCHING-PARAMETER does not feature `.short_name`...